test/test.c

```diff
@@ -153,6 +155,12 @@ static char *test_public_key_parse(apr_pool_t *pool) {
 	TST_ASSERT_ERR("oidc_jwk_rsa_bio_to_jwk", oidc_jwk_rsa_bio_to_jwk(pool, inputCert, NULL, &jwkCert, isPrivateKey, &err),
 			pool, err);
 
+	TST_ASSERT_ERR("oidc_jwk_to_json with public key", oidc_jwk_to_json(pool, jwk, &json, &err),
+			pool, err);
+	
+	TST_ASSERT_ERR("oidc_jwk_to_json with certificate", oidc_jwk_to_json(pool, jwkCert, &json, &err),
+			pool, err);
```

Comment:
shouldn't those tests also compare the return values against some hardcoded string values?

Reply:
> 
> 
> shouldn't those tests also compare the return values against some hardcoded string values?

you're completely right, I added the necessary using the ASSERT_TEST_STR